We use the fake Mozilla::CA provided by security/p5-Mozilla-CA-Fake.

Index: Build.PL
--- Build.PL.orig
+++ Build.PL
@@ -55,10 +55,10 @@ my $builder = $class->new(
                  'List::AllUtils' => 0,
                  'List::MoreUtils' => 0,
                  'List::MoreUtils::XS' => 0,
-                 'Mozilla::CA' =>  '20160104',
+                 'Mozilla::CA' => 0, 
                  'Regexp::Common' => 0,
                  'Log::Log4perl' => 0,
                  'Unicode::Collate' => '1.29',
                  'Unicode::Normalize' => '1.26',
                  'Unicode::LineBreak' => '2019.001',
                  'Unicode::GCString' => 0,
